### PR TITLE
Tagging - wafv2_web_acl add support for managing and purging tags

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

- Add support for returning tags
- Add support for updating tags
- Add support for purge_tags

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

wafv2_web_acl
wafv2_web_acl_info

##### ADDITIONAL INFORMATION

### CODE REVIEW

**Comment by tremble** [comment]
recheck

File: changelogs/fragments/1216-wafv2_web_acl-return.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+bugfixes:
+- wafv2_web_acl - consistently return web ACL info as described in module documentation (https://github.com/ansible-collections/community.aws/pull/1216).
+- wafv2_web_acl - fix ``changed`` status when description not specified (https://github.com/ansible-collections/community.aws/pull/1216).
+minor_changes:
+- wafv2_web_acl - relax botocore requirement to bare minimum required (https://github.com/ansible-collections/community.aws/pull/1216).
```
Comment (jatorcasso) [inline changelogs/fragments/1216-wafv2_web_acl-return.yml:5]: looks like this got added from your other PR :)

**Review by jatorcasso — COMMENTED** [review]
It looks like this PR contains changes intended for this PR as well as from #1216. I can give it a proper review after they're split up!

File: changelogs/fragments/1216-wafv2_web_acl-return.yml (hunk 1 above)
Comment (tremble) [reply changelogs/fragments/1216-wafv2_web_acl-return.yml:5]: Yup, you'll see two commits, unfortunately GitHub doesn't support chaining PRs in the same way it's possible to chain them in Gerrit.

**Comment by tremble** [comment]
recheck

File: plugins/modules/wafv2_web_acl.py [hunk 2]
```diff
@@ -105,6 +101,7 @@
 extends_documentation_fragment:
 - amazon.aws.aws
 - amazon.aws.ec2
+- amazon.aws.tags
```
Comment (alinabuzachis) [inline plugins/modules/wafv2_web_acl.py:105]: Do we need to document when `purge_tags` has been added with a note?

Comment (tremble) [reply plugins/modules/wafv2_web_acl.py:105]: Yup, we probably should

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**